buildpacks/maven/tests/integration/versions.rs

```diff
@@ -115,6 +115,7 @@ fn set_maven_version_app_dir_preprocessor(version: &str, path: &Path) {
         .read(true)
         .write(true)
         .create(true)
+        .truncate(false)
```

Comment:
I didn't see this in the list of lint errors, why is this changed?

Reply:
This is the lint that caused this change: https://rust-lang.github.io/rust-clippy/master/index.html#/suspicious_open_options